### DISCUSSION TITLE
Small improvement with the Checkboxes (Tree): possibility to display it open or not at the first show.

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#19037] [Feature Requests]
### Summary

TTSIA

### Motivation

The user sees the choices at one glance

### Detailed Design

Add a Toggle at the bottom of the Interface panel of the Checkboxes (Tree)
Opened: Yes/No

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

-

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
